server/background_tasks.go

```diff
@@ -647,14 +666,24 @@ func (s *Server) ResendAllDlsMsgs(stationName string, stationId int, tenantName
 		username := user.Username
 		stationIdStr := strconv.Itoa(stationId)
 
-		err := db.UpdateResendDisabledInStation(true, stationId, tenantName)
+		err := db.UpdateResendDisabledInStations(true, []int{stationId})
 		if err != nil {
-			serv.Errorf("[tenant: %v][user: %v][station: %v]ResendAllDlsMsgs at UpdateResendDisabledInStation: %v", tenantName, username, stationIdStr, err.Error())
+			errorMsg := fmt.Errorf("[tenant: %v][user: %v][station: %v]ResendAllDlsMsgs at UpdateResendDisabledInStations: %v", tenantName, username, stationIdStr, err.Error())
```

Comment:
Why not using the function you created?

Reply:
Because UpdateResendDisabledInStations did not succeed to update the resend_diabled to true so if I call to handlResendAllFailure - it do actually update the resend_disbled flag to false, but the false already in false because this function failed - so thus I didn't call to function handlResendAllFailure and just sent error message on socket. @idanasulinmemphis

Reply:
Ok but I prefer to update false value to false rather than having the same code twice so please change it